logrotate/easyengine.conf

```diff
@@ -0,0 +1,20 @@
+# logrotate config for easyengine
+# ee logs
+/opt/easyengine/logs/ee.log {
+    daily
+    missingok
+    rotate 14
+    compress
+    notifempty
+    create 0644 root root
+}
+
+# docker volume logs
+/var/lib/docker/volumes/*log*/_data/* {
```

Comment:
I think we need to be more specific about this. User might have other containers running which have log in their name. We might end up rotating their files too.

Reply:
Also, why the location of directories is same as of the host 🤔

Reply:
`/var/lib/docker/volumes/`, this directory will not work in case of macOS.